controllers/defaults.go

```diff
@@ -72,6 +79,13 @@ var (
 	NoobaaSubscriptionCatalogSource          = GetEnvOrDefault("NOOBAA_SUBSCRIPTION_CATALOGSOURCE")
 	NoobaaSubscriptionCatalogSourceNamespace = GetEnvOrDefault("NOOBAA_SUBSCRIPTION_CATALOGSOURCE_NAMESPACE")
 
+	CSIAddonsSubscriptionName                   = GetEnvOrDefault("CSI_ADDONS_SUBSCRIPTION_NAME")
```

Comment:
these variables are not used in this PR, check where `NoobaaSubscriptionName` is used, and create a subscription for csi-addons the same way